apps/console/src/features/applications/components/forms/advanced-configurations-form.tsx

```diff
@@ -231,18 +233,23 @@ export const AdvancedConfigurationsForm: FunctionComponent<AdvancedConfiguration
                 updateConfiguration(values);
             } }
         >
-            <Field.CheckboxLegacy
-                ariaLabel="Saas application"
-                name="saas"
-                label={ t("console:develop.features.applications.forms.advancedConfig.fields.saas.label") }
-                required={ false }
-                value={ config?.saas ? [ "saas" ] : [] }
-                readOnly={ readOnly }
-                data-testid={ `${ testId }-sass-checkbox` }
-                data-componentid={ `${ testId }-sass-checkbox` }
-                hint={ t("console:develop.features.applications.forms.advancedConfig.fields.saas.hint") }
-                hidden={ !applicationConfig.advancedConfigurations.showSaaS }
-            />
+            {
+                legacyAuthzRuntime &&
+                (
+                    <Field.CheckboxLegacy
+                        ariaLabel="Saas application"
+                        name="saas"
+                        label={ t("console:develop.features.applications.forms.advancedConfig.fields.saas.label") }
+                        required={ false }
+                        value={ config?.saas ? [ "saas" ] : [] }
+                        readOnly={ readOnly }
+                        data-testid={ `${ testId }-sass-checkbox` }
+                        data-componentid={ `${ testId }-sass-checkbox` }
+                        hint={ t("console:develop.features.applications.forms.advancedConfig.fields.saas.hint") }
+                        hidden={ !applicationConfig.advancedConfigurations.showSaaS }
```

Comment:
we can add the legacyAuthzRuntime check here

Reply:
Fixed with e1f4094fbb933690f99bbe2581117437e14f5879